server/graphql/vvm/src/types/vvm_status_log.rs

```diff
@@ -0,0 +1,107 @@
+use async_graphql::*;
+use chrono::NaiveDateTime;
+use repository::vvm_status::vvm_status_log_row::VVMStatusLogRow;
+use service::vvm::vvm_status_log::insert::InsertVVMStatusLogInput as ServiceInput;
+
+#[derive(PartialEq, Debug)]
+pub struct VVMStatusLogNode {
+    vvm_status_log: VVMStatusLogRow,
+}
+
+#[Object]
+impl VVMStatusLogNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn status_id(&self) -> &str {
```

Comment:
Create a loader for this please so we can get the vvm name to display in frontend